### PR TITLE
Display warning message if PYENV_ROOT is not set and abort if installed python is not the desired one

### PR DESCRIPTION
**Pull request by Perados**

### CODE REVIEW

**Comment by kennethreitz** [comment]
don't want to abort if it's the wrong python, just warn.

**Comment by kennethreitz** [comment]
PYENV_ROOT is almost always set, from what i can tell. i think you're just a special case :)

File: pipenv/cli.py [hunk 1]
```diff
@@ -390,13 +389,23 @@ def abort():
...
+                    '{0}{1}versions{1}*{1}bin'.format(
+                        PYENV_ROOT,
+                        os.sep
+                    )
+                ):
+                    add_to_path(found)
+            else:
+                click.echo(
+                    u'{0}: PYENV_ROOT is not set. New python paths will '
+                    u'probably not be exported properly after installation.'
```
Comment (kennethreitz) [inline pipenv/cli.py]: no u here

**Comment by Perados** [comment]
But then the wrong python would be used for the venv. That's not the desired behaviour, is it?

Haha well, I was only using pyenv to install different versions of python, then using the paths with virtualenvwrapper, that's why I never needed to set PYENV_ROOT.

**Comment by kennethreitz** [comment]
take out the sys.exit and i'll merge

File: pipenv/cli.py [hunk 2]
```diff
@@ -390,13 +389,23 @@ def abort():
...
+                        PYENV_ROOT,
+                        os.sep
+                    )
+                ):
+                    add_to_path(found)
+            else:
+                click.echo(
+                    '{0}: PYENV_ROOT is not set. New python paths will '
+                    'probably not be exported properly after installation.'
+                    .format(
```
Comment (kennethreitz) [inline pipenv/cli.py]: `''.format`

**Comment by Perados** [comment]
@kennethreitz Alright, I removed the sys.exit. :)

**Pull request merged**